kalite/distributed/templates/distributed/base.html

```diff
@@ -61,15 +61,13 @@
             <script type="text/javascript" src="//cdn.crowdin.com/jipt/jipt.js"></script>
         {% endif %}
 
+        {% compress js file basejs %}
         <!-- JS translation file statically included as a back-up, so that if the saved file below isn't available, we don't choke. -->
-        {% if current_language != "en" %}
```

Comment:
Yeah, this seems like the best solution, with no drawback really. Could, perhaps, put these around the script tag below?

My main interest is how these compiled tags would work at run time - it is something I am not clear on, but I assume it will pull in whichever i18n/<lang>.js file was active when we ran the compressor.